shiny-updates.php

```
@@ -628,3 +629,35 @@ function wp_ajax_search_plugins() {
 
 	wp_send_json_success( $status );
 }
+
+
+/**
+ * Ajax handler for searching plugins to install.
+ *
+ * @since 4.5.0
+ *
+ * @global WP_List_Table $wp_list_table
+ * @global string        $hook_suffix
+ */
+function wp_ajax_search_install_plugins() {
+	check_ajax_referer( 'updates' );
+
+	global $wp_list_table, $hook_suffix;
+	$hook_suffix = 'plugins.php';
+
+	$status        = array();
+	$wp_list_table = _get_list_table( 'WP_Plugin_Install_List_Table' );
+
+	if ( ! $wp_list_table->ajax_user_can() ) {
+		$status['error'] = __( 'You do not have sufficient permissions to manage plugins on this site.' );
```

We never handle that error, do we? I wonder if we should either just fail silently here or display that error somehow to the user.